CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change history for ui-finance
 
+## [1.6.0](https://github.com/folio-org/ui-finance/tree/v1.6.0) (2019-07-23)
+[Full Changelog](https://github.com/folio-org/ui-finance/compare/v1.5.0...v1.6.0)
+
+* bump up version in package.json
```

Comment:
It doesn't describe why new release has been drafted. I think the purpose why new release was drafted - updated okapi interface of funds.

Reply:
Updated